modules/cas_cache/layer_cache_management.c

```diff
@@ -1411,6 +1385,27 @@ static void _cache_mngt_remove_core_complete(void *priv, int error)
 	complete(&context->cmpl);
 }
 
+static int _cache_mngt_remove_core_is_active(ocf_cache_t cache,
```

Comment:
Given the implementation, it would make sense to rename this function to cache_mngt_check_core_active